packages/easy-coding-standard/config/set/doctrine-annotations.php

```diff
@@ -11,20 +10,11 @@
 
 return static function (ContainerConfigurator $containerConfigurator): void {
     $services = $containerConfigurator->services();
-
-    $services->set(DoctrineAnnotationNewlineInNestedAnnotationFixer::class);
-
-    $services->set(DoctrineAnnotationIndentationFixer::class)
-        ->call('configure', [[
-            'indent_mixed_lines' => true,
-        ]]);
-
+    $services->set(DoctrineAnnotationArrayAssignmentFixer::class)
+        ->call('configure', [['operator' => ':']]);
     $services->set(DoctrineAnnotationBracesFixer::class);
+    $services->set(DoctrineAnnotationIndentationFixer::class);
+    $services->set(DoctrineAnnotationNewlineInNestedAnnotationFixer::class);
     $services->set(DoctrineAnnotationSpacesFixer::class)
-        ->call('configure', [[
-            'after_array_assignments_equals' => false,
-            'before_array_assignments_equals' => false,
-        ]]);
-
-    $services->set(DoctrineAnnotationArrayAssignmentFixer::class);
+        ->call('configure', [['before_array_assignments_colon' => false]]);
```

Comment:
Also, this is not PHP CS Fixer set, but ECS one. I'll revert it, to prevent unintended coding standard breaks.

To make PHP CS Fixer one, a `PHP_CS_FIXER_` prefix could be added.